src/java/org/apache/cassandra/cql3/statements/schema/AlterKeyspaceStatement.java

```diff
@@ -62,6 +63,9 @@ public AlterKeyspaceStatement(String keyspaceName, KeyspaceAttributes attrs)
 
     public Keyspaces apply(Keyspaces schema)
     {
+        if (ClientWarn.instance.get() == null)
+            ClientWarn.instance.captureWarnings();
+        int previousSize = ClientWarn.instance.getWarnings() == null ? 0 : ClientWarn.instance.getWarnings().size();
```

Comment:
Nit: maybe we could name this using the `warnings` word, something like `previousNumWarnings`, or `previousWarningsSize`? Feel free to ignore if you don't agree.